client/components/payment-activity/style.scss

```diff
@@ -1,15 +1,55 @@
 .wcpay-payments-activity {
 	&__card {
-		padding: 24px 24px 16px 24px;
-
 		&__header {
-			padding: 0 !important;
-			margin-bottom: 32px;
+			padding: 16px 24px !important;
+			font-size: 20px;
+			font-weight: 400;
 		}
 
 		&__body {
 			text-align: center;
-			color: $studio-gray-30;
+			padding: 0 !important;
+		}
+	}
+}
+
+.payments-activity-data {
+	display: flex;
+	padding: 24px;
+	border-top: 1px solid $gray-200;
+	border-bottom: 1px solid $gray-200;
+
```

Comment:
We will need to add a breakpoint for mobile view. 

```suggestion
@include breakpoint( '<660px' ) {
		flex-direction: column;
	}

```

#### Before change: 

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/d00ac581-2d0d-4304-a490-82a186a76c55)


#### After change:

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/0fe501f2-f6ec-46a4-9eff-4e7f332693bb)

Reply:
Thanks @nagpai. Implemented!

Reply:
This looks great now, Thank you!